docs/site/decorators/Decorators_openapi.md

```diff
@@ -191,6 +191,34 @@ You can find specific use cases in
 _The parameter location cookie is not supported yet, see_
 _(https://github.com/strongloop/loopback-next/issues/997)_
 
+### Parameter decorator for objects
+
+One of the parameter decorator types, `@param.query.object` is applied to
+generate an OpenAPI definition for query parameters with JSON values.
+
+The generated definition supports the `url-encoded` style specified in Open-API.
+This style supports receiving url-encoded payload for json query parameters.
+
+For instance, to filter api results from the GET '/todo-list' endpoint in the
```

Comment:
Same comment in #4620, maybe just `to filter the results...` (without `api`) ?

Reply:
agreed